test/elixometer_test.exs

```diff
@@ -82,7 +82,16 @@ defmodule ElixometerTest do
 
   def subscription_exists(metric_name) when is_list(metric_name) do
     wait_for_messages()
-    metric_name in Reporter.subscriptions
+    metric_name in Reporter.subscription_names
+  end
+
+  def subscription_exists(metric_name, datapoint) when is_bitstring(metric_name) do
```

Comment:
Usually these sort of checks (when don't change the state of the system and return a boolean) would have a `?` at the end,  i.e. `subscription_exists?`. It is just a convention so not required.